ios/Scripts/generate_app_icons.py

```
@@ -0,0 +1,173 @@
+
+
+def create_pulseplate_icon(size: int) -> Image.Image:
+    """Создает иконку PulsePlate с заданным размером
+
+    Валидация размера предотвращает деление на ноль и некорректные расчеты
+    для слишком маленьких, отрицательных и нецелочисленных значений.
+    """
+    # Early validation for size to avoid malformed drawing or ZeroDivision errors
+    try:
+        size = size
+    except (TypeError, ValueError) as exc:  # noqa: F841 - exc kept for clarity
```

Ruff (0.13.1)

Unused `noqa` directive (unused: `F841`)

Remove unused `noqa` directive

(RUF100)